src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -514,11 +515,19 @@ export class DeploymentsService implements OnDestroy {
     return this.isApplicationDeployedInEnvironment(spaceId, applicationId, environmentName)
     .flatMap((deployed: boolean) => {
       if (deployed) {
-        const url = `${this.apiUrl}${spaceId}/applications/${applicationId}/deployments/${environmentName}/statseries?start=${startTime}&end=${endTime}`;
-        return this.http.get(url, { headers: this.headers })
+        const key = `${spaceId}:${applicationId}:${environmentName}`;
+        if (!this.multiTimeseriesObservables.has(key)) {
+          const subject = new ReplaySubject<MultiTimeseriesData>(1);
+
+          const url = `${this.apiUrl}${spaceId}/applications/${applicationId}/deployments/${environmentName}/statseries?start=${startTime}&end=${endTime}`;
+          const observable = this.http.get(url, { headers: this.headers })
           .map((response: Response) => (response.json() as MultiTimeseriesResponse).data)
```

Comment:
I think these lines need to be indented one level past the observable declaration. Looks good otherwise

Reply:
Ah thanks for catching that, I'll indent it and re-push it.